src/Core/Extensions/Http.cs

```diff
@@ -23,9 +23,14 @@ internal static async Task ExecuteHttpProxyOperationAsync(this HttpContext conte
                     .GetService<IHttpClientFactory>()
                     .CreateClient(options?.HttpClientName ?? Helpers.HttpProxyClientName);
 
+                if (options?.Filter != null && !options.Filter(context))
+                {
+                    return false;
```

Comment:
We can probably keep this, but I feel like, now that there are two filter options, this may call for an enum to be more clear than a `bool`.  I might just clean it up after, and then bump a major version.

Reply:
You're right, I wanted to add it with boolean because it would be the simplest implementation. But the code is much more confusing because of that, an Enum absolutely solves this. I'll try to make a change later today.